.gitignore

```diff
@@ -53,3 +53,4 @@ celerybeat-schedule
 /locale/*/*/*.mo
 .grunt
 node_modules/
+/BLOB_FILES/
```

Comment:
these aren't just blob files so i think the name is a bit misleading. maybe like `SHARED_DRIVE`